transformations/contraction_expansions/transformation.py

```diff
@@ -0,0 +1,227 @@
+            "i've": "i have",
+            "isn't": "is not",
+            "it'd": "it would",
+            "it'll": "it will",
+            "it's": "it is",
+            "ma'am": "madam",
+            "might've": "might have",
+            "mightn't": "might not",
+            "must've": "must have",
+            "mustn't": "must not",
+            "needn't": "need not",
+            "oughtn't": "ought not",
+            "shan't": "shall not",
+            "she'd": "she would",
+            "she'll": "she will",
+            "she's": "she is",
+            "should've": "should have",
+            "shouldn't": "should not",
+            "that'd": "that would",
+            "that's": "that is",
+            "there'd": "there would",
+            "there's": "there is",
+            "they'd": "they would",
+            "they'll": "they will",
+            "they're": "they are",
+            "they've": "they have",
+            "wasn't": "was not",
+            "we'd": "we would",
+            "we'll": "we will",
+            "we're": "we are",
+            "we've": "we have",
+            "weren't": "were not",
+            "what're": "what are",
+            "what's": "what is",
+            "when's": "when is",
+            "where'd": "where did",
+            "where's": "where is",
+            "where've": "where have",
+            "who'll": "who will",
+            "who's": "who is",
+            "who've": "who have",
+            "why's": "why is",
+            "won't": "will not",
+            "would've": "would have",
+            "wouldn't": "would not",
+            "you'd": "you would",
+            "you'd've": "you would have",
+            "you'll": "you will",
+            "you're": "you are",
+            "you've": "you have",
+        }
+
+        contraction_pattern = re.compile(
+            r"\b({})\b".format("|".join(contraction_map.keys())),
+            flags=re.IGNORECASE | re.DOTALL,
+        )
+
+        def expand_match(contraction):
+            match = contraction.group(0)
+            first_char = match[0]
+            expanded_contraction = contraction_map.get(
+                match, contraction_map.get(match.lower())
+            )
+            expanded_contraction = first_char + expanded_contraction[1:]
+            return expanded_contraction
+
+        return contraction_pattern.sub(expand_match, sentence)
+
+    def contractions(self, sentence):
+        """Perturbation functions, contracts and expands contractions if present
+        Parameters
+        ----------
+        sentence : str
+            input
+        Returns
+        -------
+        list
+            List of strings with contractions expanded or contracted, or []
+        """
+        expanded = [self.expand_contractions(sentence), self.contract(sentence)]
+        return [t for t in expanded if t != sentence]
+
+    def generate(self, sentence):
+        pertubed = self.contractions(sentence)[0]
+        return pertubed
```

Comment:
Please pull the main into your branch.

The returned texts should be a list now. You can either wrap up the perturbed, like `[perturbed]`, or returning multiple outputs in a list (e.g. both expanded and contracted version).

Don't forget to update your test.json accordingly.